src/containers/WelcomePage/components/TableComponent.tsx

```diff
@@ -0,0 +1,183 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import styled from '@emotion/styled';
+import { spacing, colors, fonts } from '@ndla/core';
+import { ReactNode } from 'react';
+import { ExpandLess, ExpandMore } from '@ndla/icons/action';
+import { css } from '@emotion/react';
+import isEmpty from 'lodash/isEmpty';
+import Spinner from '../../../components/Spinner';
+
+const StyledTable = styled.table`
+  font-family: arial, sans-serif;
+  border-collapse: separate;
+  width: 100%;
+  border-spacing: 0;
+  font-family: ${fonts.sans};
+  margin-bottom: 0px;
+  table-layout: fixed;
+  min-width: 850px;
+  display: inline-table;
+  th {
+    font-weight: ${fonts.weight.bold};
+    padding: 0px ${spacing.xsmall};
+    border-bottom: 1px solid ${colors.text.primary};
+    background-color: ${colors.brand.lighter};
+  }
+  th:not(:first-of-type) {
+    border-left: 1px solid ${colors.text.primary};
+  }
+  td {
+    ${fonts.sizes(16, 1.1)};
+    padding: ${spacing.xsmall} ${spacing.xsmall};
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+  }
+  tr {
+    height: 30px;
+  }
+  tr:nth-of-type(odd) {
+    background: ${colors.brand.greyLightest}80;
```

Comment:
Det her var ganske magisk 🧙 

Vi burde egentlig hatt en egen farge for dette, men det er nok mange steder vi mangler farger atm. Trenger nok ikke å se på det nå

Reply:
Tidligere hadde vi feks transparent rød på hvit bakgrunn i stedet for egen fargekode for lys rød. Men vi har gått bort fra transparens fordi det blir feil når bakgrunnen ikke er hvit.

Jeg vet ikke helt hva planen er for fargesystem vårt, men ideelt sett hadde vi hatt denne fargen tilgjengelig uten transparens. Lekte meg litt med fargene vi har tilgjengelig. Kunne vi feks brukt `colors.brand.lightest`